.github/workflows/run_tests.yaml

```diff
@@ -0,0 +1,55 @@
+name: Lint and Test
```

Comment:
These can be uncommented once we have the schema and some sample data in place.

The pip commands can be converted to uv once that PR has been merged.

Reply:
Skip this file now that schema is in another repo?